WORKSPACE

```diff
@@ -0,0 +1,73 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+http_archive(
+    name = "io_bazel_rules_go",
+    sha256 = "a8d6b1b354d371a646d2f7927319974e0f9e52f73a2452d2b3877118169eb6bb",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/v0.23.3/rules_go-v0.23.3.tar.gz",
+        "https://github.com/bazelbuild/rules_go/releases/download/v0.23.3/rules_go-v0.23.3.tar.gz",
+    ],
+)
+
+git_repository(
+    name = "deploymentmanager-autogen",
+    branch = "master",
```

Comment:
Should we pin to a commit/tag instead? My thinking is that having a moving source like this will cause the builds to be not hermetic.

Reply:
That's a good point, the only reason I didn't is we have no tags yet. Will add one